esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -72,6 +73,7 @@ and ``opt`` directories within the workflow. The files required are:
 
 ``opt/rose-suite-<site>.conf``
   Contains configuration items specific to the ``SITE``, including ``SITE``
+  and paths specific to the ``SITE`` on where to find the required datasets
```

Comment:
To minimise the duplication, how about:
```suggestion
  and where to find the required datasets
```
?

Reply:
d27908d58d77fb2da75bbe129c2381e895071e1e